coderd/coderd.go

```diff
@@ -319,26 +319,12 @@ func New(options *Options) *API {
 			// Specific routes can specify smaller limits.
 			httpmw.RateLimit(options.APIRateLimit, time.Minute),
 		)
-		r.Get("/", func(w http.ResponseWriter, r *http.Request) {
-			httpapi.Write(r.Context(), w, http.StatusOK, codersdk.Response{
-				//nolint:gocritic
-				Message: "👋",
-			})
-		})
+		r.Get("/", apiRoot)
```

Comment:
Had the idea, what if we would write the `@Summary ...` tags here, for the routes, instead of the handlers? Neither is right or wrong per-se, both are only half the truth, but one handler could be used in multiple routes, for instance.

Reply:
I tried this approach, but comments are not "discovered" then.

Reply:
Ah, I feared that may be the case, that's unfortunate. We'll keep going as we have, then. 😄